apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -24,10 +27,12 @@ export function Error({
         </h1>
         <p className="text-lg text-contrast-500">{subtitle}</p>
 
-        {cta !== undefined && cta.href !== '' && cta.label !== '' && (
-          <ButtonLink className="mt-8" href={cta.href} size="large" type="button" variant="primary">
-            {cta.label}
-          </ButtonLink>
+        {ctaLabel !== '' && (
```

Comment:
I don't think we should check for the empty string here. If somebody explicitly passes that, it's kind of on them if we render a button with an empty label